### DISCUSSION TITLE
How to get grid to show up when working with Grid Layout in JPanel?

### DISCUSSION THREAD

**Discussion by ForceConstant** [eclipse/windowbuilder#582] [Q&A] [answered]
So I drag a JPanel onto my GUI, and Set the layout to Grid. I then set in properties the column to 2, and rows to 2. But when I then drag a JLabel onto the panel, I do not see the grid, why not? What else do I need to do?

**Comment by ptziegler** [comment, marked as answer]
The grid is only shown for the _GridBagLayout_, not the _GridLayout_.

**Reply by ForceConstant** [reply]
Ahh that is too bad.
